### ISSUE TITLE
Changing the FOV of the Lidar in ROS

### ISSUE THREAD

**Issue by Vineet-Pandey** [ouster-lidar/ouster-ros#227]
I want to change the FOV of the Ouster OS0-32 from [0-360] to [0, 180000] and in the Software User guide i see  the use of ``` set_config_param azimuth_window``` How can i set this param in ros? I don't see this particular rosparam in rosparam list.

**Platform :**
- Ouster Sensor: OS-0-32
- ROS version/distro Melodic
- Operating System? Ubuntu 18

**Comment by Samahu** [comment]
We don't expose the ability to set azimuth window via ros launch file. We have `/ouster/set_config/` (described [here](https://github.com/ouster-lidar/ouster-ros#setconfig-experimental) which could be used for this purpose but due to sensor re-initialization this is unlikely to work for ROS1 driver. It does work though for the ROS2 implementation since it implements Node LifeCycle.

**Comment by imdsafi09** [comment]
Hi @Samahu 
Could you please further elaborate how we can change the configuration to set the azimuth window up to 180 ° in ROS-2?

**Comment by Samahu** [comment]
Sure, create a config file in some known path and add the following content:
```json
{
    "azimuth_window": 
    [
        0,
        180000
    ],
}
```
This config would set the azimuth window to 180 degrees. Let's name the file os-xxx.json. Now to apply this config file you can do so by invoking the command:
```bash
ros2 service call /ouster/set_config ouster_sensor_msgs/srv/SetConfig "{config_file: /path/to/file/os-xxx.json}"
```
Make sure to set the correct path of your file. If the call was successful you should get the following response:
```bash
ouster_sensor_msgs.srv.SetConfig_Response(config='{\n    "azimuth_window": \n    [\n        0,\n        180000\n    ]\n}')
```
The driver will stop momentarily and and restart with new the new azimuth window configuration.

**Comment by imdsafi09** [comment]
Hi @Samahu,

Thank you for your response. I would like to know whether the above configuration will only limit the field of view of the point cloud or image information  (signal image, etc.) as well?


Best regards,

**Comment by Samahu** [comment]
It will affect both

**Comment by Samahu** [comment]
addressed in #357 and #358
